src/zenml/integrations/aws/step_operators/sagemaker_step_operator.py

```diff
@@ -272,7 +272,7 @@ def launch(
                 "ExperimentName": settings.experiment_name,
                 "TrialName": sanitized_training_job_name,
             }
-        info.force_write_logs()
+
```

Comment:
Can you explain why this isn't needed anymore?

Reply:
For the time being, I am going to implement a flush functionality and add this method back.

Reply:
I like the different log sources, I think that would solve the issue. For now, removing this would cause the following, which I think was also part of this initial ticket:
- step container logs a few messages, but they don't get flushed
- step container launches the step operator container
- step operator container logs enough messages to flush. These now get written before the logs in the initial bullet point.

Reply:
After switching to single queue, I still need to adjust it. I am currently on it.